playbooks/os/openshift-cluster/vars.yml

```diff
@@ -0,0 +1,33 @@
+---
+openstack_network_prefix: 'openshift-ansible-{{ cluster_id }}'
```

Comment:
These variables should be override-able using env variables.  It's not a big deal for the libvirt provider, since it is used  for local instance creation, but for providers that can be used to host production instances, we should allow more flexibility to the user without having to overwrite the vars files.